integration/tests/basic/test_batch_operations.py

```diff
@@ -0,0 +1,162 @@
+import random
+
+import pytest
+import requests
+from semantic_version import Version
+from web3 import Web3
+
+from integration.tests.basic.helpers import rpc_checks
+from utils.accounts import EthAccounts
+from utils.apiclient import JsonRPCSession
+from utils.web3client import NeonChainWeb3Client
+
+
+@pytest.mark.usefixtures("accounts", "web3_client")
+class TestBatchOperations:
+    accounts: EthAccounts
+    web3_client: NeonChainWeb3Client
+
+    def test_batch_operations_same_function(
+            self,
+            request: pytest.FixtureRequest,
+            common_contract,
+            json_rpc_client: JsonRPCSession,
+    ):
+        """
+        sends a batch operation request with 100 calls to the same function with different parameters
+        """
+        account = self.accounts[0]
+        evm_version: Version = self.web3_client.get_evm_version_number()
+        data_or_input_field_name = "data" if evm_version < Version("1.13.0") else "input"
+
+        batch = []
+        expected_results = []
+
+        for id_ in range(1, 101):
```

Comment:
how long it takes to run this test?Is it still fast if there 100 requests? Just curious

Reply:
2.5 sec excluding setup